network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/app_install_hero.html

```diff
@@ -0,0 +1,63 @@
+{% load i18n wagtailcore_tags wagtailimages_tags %}
+
+<!-- {% image page.hero_background original as hero_background %} -->
```

Comment:
Could remove stray comment while in here.

Reply:
Great catch @robdivincenzo! This has been removed